submissions/AriadnaKravchuk/memory-game/style.css

```diff
@@ -0,0 +1,448 @@
+}
+
+.game__status-board {
+    width: fit-content;
+    gap: var(--status-board-gap);
+    font-size: var(--status-board-font-size);
+    font-family: inherit;
+    background-color: var(--status-board-background);
+    border: var(--game-page-border-settings);
+    border-radius: var(--status-board-radius);
+    padding: var(--status-board-padding);
+    margin: var(--status-board-margin);
+}
+
+.popup {
+    position: fixed;
+    top: 0;
+    left: 0;
+    background-color: rgba(245, 245, 245, 50%);
+}
+
+.popup__inner {
+    display: flex;
+    width: calc(100% - 2.250rem);
+    padding: 1.563rem 1.125rem;
+    max-width: var(--popup-width);
+    align-items: center;
+    color: black;
+    background-color: white;
+    border-radius: 1.25rem;
+}
+
+.popup__title {
+    font-size: var(--popup-title-font-size);
+    margin: 1.25rem 0;
+}
+
+.popup__text {
+    font-size: var(--popup-text-font-size);
+}
+
+.popup__buttons {
+    display: flex;
+    justify-content: center;
+    align-items: stretch;
+    margin-top: 2.5rem;
+    width: 95%;
+    gap: 5%;
+}
+
+.popup__button {
+    max-width: 150px;
+    padding: var(--popup-button-padding);
+    font-size: var(--popup-button-font-size);
+    color: white;
+    font-family: inherit;
+    background-color: var(--popup-button-color);
+    border-radius: 5px;
+}
+
+.popup__button:hover,
+.popup__button:active {
+    background-color: var(--popup-button-color-hover);
+}
+
+:root body.gothic {
+    --popup-button-color: #32312D;
+    --game-page-background: whitesmoke;
+    --game-page-font-color: #32312D;
+    --game-page-font-family: 'Rye', cursive;
+    --game-page-board-background: ivory;
+    --card-front-background: #32312D;
+    --card-back-background: whitesmoke;
+    --status-board-background: ivory;
+    --game-page-border-settings: 0.313rem #32312D solid;
+    --card-front-border: #20201e;
+    --card-back-border: lightgrey;
+    --card-front-color: ivory;
+    --popup-button-color-hover: #3b3a35;
+}
+
+:root body.daily-life {
+    --popup-button-color: #45B649;
+    --game-page-background: #DCE35B;
+    --game-page-font-color: white;
+    --game-page-font-family: 'Capriola', sans-serif;
+    --game-page-board-background: beige;
+    --card-front-background: #45B649;
+    --card-back-background: #DCE35B;
+    --status-board-background: #45B649;
+    --card-front-border:#48a34b;
+    --card-back-border: #d2d85a;
+    --card-front-color: white;
+    --popup-button-color-hover: #46a049;
+}
+
+:root body.sailor-moon {
+    --popup-button-color: #bba6c1;
+    --game-page-background: #dbc5db;
+    --game-page-font-color: white;
+    --game-page-font-family: 'Gotu', sans-serif;
+    --game-page-board-background: beige;
+    --card-front-background: #bba6c1;
+    --card-back-background: #dbc5db;
+    --status-board-background: #bba6c1;
+    --card-front-border:#c7bac7;
+    --card-back-border: #d1c1d1;
+    --card-front-color: white;
+    --popup-button-color-hover: #a494a8;
+}
```

Comment:
Please avoid using tag selectors, it is better to use class or id  selectors. This is reason why  >>> [article](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css).
Is `body.sailor-moon` redundant? Can we leave only `.sailor-moon` selector?